src/controllers/order-controller.js

```diff
@@ -64,8 +65,33 @@ async function createOrder(req, res, next) {
   }
 }
 
+async function updateDeliveryStatus(req, res, next) {
+  const { orderId } = req.params;
+
+  try {
+    const findQuery = { _id: orderId };
+    const order = await OrderModel.findOne(findQuery).select('ownerId');
```

Comment:
add ownerId: req.userData._id to db find query, instead of doing it on code level